packages/flutter/lib/src/material/choice_chip.dart

```diff
@@ -158,7 +165,7 @@ class ChoiceChip extends StatelessWidget
       onSelected: onSelected,
       pressElevation: pressElevation,
       selected: selected,
-      showCheckmark: Theme.of(context).useMaterial3,
+      showCheckmark: showCheckmark ?? Theme.of(context).useMaterial3,
       tooltip: tooltip,
```

Comment:
```suggestion
      checkmarkColor: checkmarkColor,
      tooltip: tooltip,
```

Pass the `ChoiceChip.checkmarkColor` to the `RawChip` you added. 
We don't need to pass the theme value here since `RawChip` will handle it.